index.html

```diff
@@ -0,0 +1,119 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nature UI</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel = "icon" href = "./gardenin-icon.png" type = "image/x-icon"> 
+    <link href="styles.css" type="text/css" rel="stylesheet"/>
+	<style>
+		body{
+			height: 100vh;
+			display: grid;
+			grid-template-columns: 3fr 2fr;
+		}
+
+	</style>
+</head>
+<body>
+    <nav class="navbar">
+			<div class="container-center">
+				<div class="hamburger-menu pointer" role="button" tabindex="0">
+					<span class="hamburger-menu-line"></span>
+					<span class="hamburger-menu-line"></span>
+					<span class="hamburger-menu-line"></span>
+                </div>
+
+				<a href="/" class="container-center brand-logo">
+					<p class="brand-name">Nature <span>UI</span></p>
+				</a>
+			</div>
+			<div class="container-center">
+				<a href="/" class="flex">
+					<!-- <img src="./nature.png" alt="avatar" class = "circle-border avatar-img sm"> -->
+					Github
+					<i
+						style="font-size: 1.5rem; margin-left: 0.2rem"
+						class="fa fa-github"
+						aria-hidden="true"
+					></i>
+				</a>
+			</div>
+
+			<ul class="menu hide-menu" id="menu">
+				<li class="menu-item">
+					<b><a href="index.html">Home</a></b>
+				</li>
+				<li class="menu-item">
+					<a href="./components.html">Installation</a>
+				</li>
+				<li class="menu-item">
+					<b><a href="./components.html#alerts">Components</a></b>
+				</li>
+				<li class="menu-item">
+					<a href="./components.html#alerts">Alerts</a>
+				</li>
+				<li class="menu-item">
+					<a href="./components.html#buttons">Buttons</a>
+				</li>
+				<li class="menu-item">
+					<a href="./components.html#avators">Avatar</a>
+				</li>
+				<li class="menu-item">
+					<a href="./components.html#listgroups">List Groups</a>
+				</li>
+				<li class="menu-item">
+					<a href="./components.html#badges">Badges</a>
+				</li>
+				<li class="menu-item">
+					<a href="./components.html#productcards">Product Card</a>
+				</li>
+				<li class="menu-item">
+					<a href="./components.html#textcards">Text Card</a>
+				</li>
+				<li class="menu-item">
+					<a href="./components.html#progress">Progress</a>
+				</li>
+				<li class="menu-item">
+					<a href="./components.html#forms">Forms</a>
+				</li>
+				<li class="menu-item">
+					<a href="./components.html" aria-disabled="true">Examples</a>
+				</li>
+				<li class="hide-in-side-nav menu-item">
+					<a href="/" class="flex">
+						<!-- <img src="./nature.png" alt="avatar" class = "circle-border avatar-img sm"> -->
+						Github
+						<i
+							style="font-size: 1.5rem; margin-left: 0.2rem"
+							class="fa fa-github"
+							aria-hidden="true"
+						></i>
+					</a>
+				</li>
+			</ul>
+		</nav>
+		<div style="padding: 3rem; padding-top: 6rem; display: flex; flex-direction: column; justify-content: center;">
```

Comment:
Instead of giving `div` make use of more Semantic HTML tags, you can give `section` and wrap all inside of that to help improve the readability of the code/markup.

Reply:
Sure I will do it. All other changes are done. Thanks for reviewing.